packages/flutter/test/material/chip_test.dart

```diff
@@ -1630,6 +1630,47 @@ void main() {
       semanticsTester.dispose();
     });
 
+    testWidgets('delete', (WidgetTester tester) async {
+      final SemanticsTester semanticsTester = SemanticsTester(tester);
+
+      await tester.pumpWidget(MaterialApp(
+        home: Material(
+          child: RawChip(
+            label: const Text('test'),
+            onDeleted: () { },
+          ),
+        ),
+      ));
+
+      expect(semanticsTester, hasSemantics(
+          TestSemantics.root(
+            children: <TestSemantics>[
+              TestSemantics(
+                textDirection: TextDirection.ltr,
+                children: <TestSemantics>[
+                  TestSemantics(
+                    flags: <SemanticsFlag>[SemanticsFlag.scopesRoute],
+                    children: <TestSemantics>[
+                      TestSemantics(
+                        label: 'test',
+                        textDirection: TextDirection.ltr,
+                        children: <TestSemantics>[
+                          TestSemantics(
```

Comment:
Shouldn't this look more like a button semantics test? Like https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/flat_button_test.dart#L490

Reply:
Updated to use button flag that reflects the change made in the actual Semantics